### ISSUE TITLE
Missing /bin/ toplevel area upon installing wmagent pypi package 

### ISSUE THREAD

**Issue by todor-ivanov** [dmwm/WMCore#11584]
**Impact of the bug**
 Python based deployment

**Describe the bug**
Similarly to https://github.com/dmwm/WMCore/issues/11583 there is yet another top level directory missing in the pypi installation. This time the directory is `/bin` , but this time the directory is actually present in the pypi package: [1], but is not install with pip. The bellow command returns no result:
```
(WMAgent.dock) [cmst1@unit01:/data/srv/wmagent/current]$ find / -name adhoc-scripts
``` 

[1]
```
(WMCore.venv3) [user@unit02 wmagent-2.2.0.5]$ tree -L 2
.
├── bin
│   ├── acdcserver-tools
│   ├── adhoc-scripts
│   ├── attempt-to-patch.sh
...
│   ├── wmcore-new-config
│   └── wmcore-new-flow
├── LICENSE
├── MANIFEST.in
├── NOTICE
├── PKG-INFO
├── README.md
├── requirements.txt
├── setup_build.py
├── setup.cfg
├── setup_dependencies.py
├── setup.py
└── src
    ├── couchapps
    ├── css
    ├── javascript
    ├── python
    └── templates
``` 

**How to reproduce it**
Setup the wmagent pypi package with:   `pip install wmagent==2.2.0.2` and try to find the WMCore bin directory or anything from its contents.
 
**Expected behavior**
Everything included in the pypi package to get deployed at the destination 

**Additional context and error message**
This together with the already mentioned issue are blockers for completing the python based deployment for the wmagent package:

https://github.com/dmwm/WMCore/issues/8797

**Comment by amaltaro** [comment]
@todor-ivanov I think it would make more sense to actually incorporate this content into the other ticket that you opened today, this:
https://github.com/dmwm/WMCore/issues/11583

Given that it's the same package and same development, just a different directory, can you please update the title and description of #11583 and close this ticket? Thanks

**Comment by todor-ivanov** [comment]
Hi @amaltaro , I agree we can merge those. Even though it would have been better to resolve them separately. And here is my motivation why I created  them as two issues, rather than a single one. 
* Because one of them has more severe impact than the other - The lack of `/etc` in the package concerns only the missing agent config templates, which are a matter of a single download from source at build time for the docker image. While the lack of `/bin` in the package is a blocker, because it is preventing any `$manage` command to be executed at the agent and is a matter of several files rather than a single template. 
* Then,  the solutions of them both (if I am not wrong) must happen at different places:
   * for `/etc`  in `setup_build.py`
   * for `/bin/ in `setup.py`

Anyway, since I already implemented a nasty workaround in deployment repo (which BTW must go away ASAP, because it is really NOT the way how we should do this), I am willing to merge these issues here ins a single one as you suggested.   

The issue: https://github.com/dmwm/WMCore/issues/11583 has been updated to explain the two different use cases and I am closing this one.
